demo/dynamic_embedding/movielens-1m-keras-notebook/movielens_dynamic_embedding_tutorial.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.                        int(self.model.lookup_vocab_sizes[k]*0.8), 
</pre><p>The <code>num_reserved</code> looks like too low, which may hurt the AUC significantly, may we change it to <code>0.95</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/recommenders-addons/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Hey @rhdong,

On this, it takes about an hour (if not more) for me to run this end to end (as I need to generate graphs with comparisons). I think it's fine to have a lower accuracy with restrict. 

The way I see, we have proven that `dynamic_embedding` can reach competitive performance with standard `Embedding` when not using `restrict()`. So I think it's okay to have sub-optimal performance using `restrict()` at `0.8`. The goal of the section using `restrict()` wasn't to reach good performance, but to show the effect of `restrict()` functionality. I'm happy to highlight this in the blog if you think that's better. The readers should be able to interpolate between the lines and understand the higher the `num_reserved` the better. I think that's just an implementation detail we don't have to worry too much? 

What do you think?

Reply:
Yeah, highlighting it makes sense, thank you so much!